application/controllers/Cadet.php

```diff
@@ -144,9 +144,9 @@ function uploadpic()
         
         $config['upload_path']      = 'images/';
         $config['allowed_types']    = 'jpeg|jpg|png';
-        $config['max_size']         = 2048;
```

Comment:
Definitely don't make this all 0 because then a cadet can't upload their photo

Reply:
uploadwingpic() should be correct now